django/db/models/base.py

```diff
@@ -370,15 +370,22 @@ def _default_manager(cls):
         return cls._meta.default_manager
 
 
+class ModelStateFieldsCacheDescriptor:
+    def __get__(self, instance, cls):
```

Comment:
Not sure, discussion is https://groups.google.com/d/topic/django-developers/3eSHhZUzxv4/discussion.